packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -96,14 +107,46 @@ export const borderAndBoxShadowStyle = (props: {
   hasErrorMessage: boolean;
   disabled: boolean;
 }): string => {
-  const { theme, hasErrorMessage, disabled } = props;
-  const borderColor = hasErrorMessage ? theme.semanticColors.errorText : theme.palette.neutralSecondary;
+  return mergeStyles(borderEditBoxStyle({ ...props, defaultBorderColor: props.theme.palette.neutralSecondary }));
+};
+
+/**
```

Comment:
Should we have a richTextSendBox Styles file?

Reply:
It's not for send box per se but for the border component that's inside of input box component. Moved it to `RichTextEditor.styles` file